packages/kbn-grid-layout/grid/grid_layout.tsx

```diff
@@ -127,13 +127,10 @@ export const GridLayout = ({
         layoutRef.current?.classList.remove('kbnGrid--mobileView');
       }
 
-      if (currentExpandedPanelId) {
-        layoutRef.current?.classList.add('kbnGrid--static', 'kbnGrid--hasExpandedPanel');
-      } else if (currentAccessMode === 'VIEW') {
+      if (currentAccessMode === 'VIEW' || currentExpandedPanelId) {
```

Comment:
what if we remove gridLayoutStateManager.expandedPanelId$ subscription
```suggestion
        if (currentAccessMode === 'VIEW') {
```

and then in drag and resize handle do this:
```
         .kbnGrid--static &,
        .kbnGridPanel--expanded & {
          display: none;
        }
```

Reply:
Good call! Done in 68ae2573775693afcae37b8be5a9c4ec8607e633